experimental/Schemes/SpecOpen.jl

```diff
@@ -378,7 +402,7 @@ The ring ``R = 𝒪(X, U)`` belongs to a sheaf of rings ``𝒪(X, -)`` and this
 the scheme ``X`` on which ``𝒪`` is defined.
 """
 scheme(R::SpecOpenRing) = R.scheme
-gens(R::SpecOpenRing) = R.(gens(base_ring(OO(scheme(R)))))
+gens(R::SpecOpenRing) = error("not implemented")
```

Comment:
```suggestion
gens(R::SpecOpenRing) = R.(gens(ambient_ring(scheme(R))))
```
Why would that go away?

Reply:
The name gens is odd. They are not generators of this k-Algebra (it may not even be finitely generated). 
They are merely coordinate functions. So maybe we should choose that name.

Reply:
Yes, they are not generators, but rather coordinates. But if we choose this other name all of a sudden while the user is used to writing 'gens' for this, it will also be confusing. Besides: gens is already used for localizations and coordinates is taken by the modules. So I would leave it with my suggestion for now.

Reply:
Well as a user I can tell you that I expect gens to return generators (in some reasonable sense). 
Besides you never used it. So I would rather leave it undefined for now.

Reply:
but `coordinates(::AbsSpec)` would make sense?